newsroom/wire/search.py

```diff
@@ -271,6 +295,28 @@ def has_permissions(self, item, ignore_latest=False):
         except Forbidden:
             return False
 
+    def get_time_filters(self) -> List[TimeFilter]:
+        """Retrieve the time filters from app config."""
+        return app.config.get("WIRE_TIME_FILTERS", [])
+
+    def get_date_filter_query(self, filter_name: str) -> Optional[DateRangeQuery]:
+        """Get the query for the given filter name."""
+        time_filters = self.get_time_filters()
+        for time_filter in time_filters:
+            if time_filter["filter"] == filter_name:
+                query = time_filter["query"]
+                query["time_zone"] = app.config.get("DEFAULT_TIMEZONE")
```

Comment:
it should use probably `query.setdefault("time_zone", app.config.get("DEFAULT_TIMEZONE"))` if it's possible to specify it also via config